plugins/woocommerce/client/admin/client/wp-admin-scripts/email-editor-integration/sidebar_settings.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { NAME_SPACE } from './constants';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Could you please replace this with @ts-expect-error? This will give us a notification when the error is fixed e.g., via updating a package, etc. Also, please provide a basic comment on why we are accepting the error.